### PR TITLE
fixed about page to new design

### PR DESCRIPTION
**Pull request by mubarakshow**
I've given the about page a new look. 

Note that there are still a few issues that need to be addressed:  
1. Page not fully responsive 
2. Links are not redirecting externally.  
3. The gap between the cards are not exactly like the design, but this is what I could do, for now. 
4. There's still background image, but that's because the [fix to this issue](https://github.com/zero-to-mastery/mappypals/pull/76) is yet to be merged. 

If everything looks fine other than these 4 issues, please proceed to merge. We'll create tasks for the issues.

### CODE REVIEW

**Comment by ArnasDickus** [comment]
I try to review it

File: src/pages/About/About.js [hunk 1]
```diff
@@ -7,37 +8,33 @@ import styled from 'styled-components';
...
-  font-size: 27px;
-  text-align: center;
-  width: 80vw;
-  height: 10rem;
-  padding: 20px;
-  margin-left: auto;
-  margin-right: auto;
-  margin-bottom: 50px;
-`
+// const WhoWeAre = styled.div`
```
Comment (ArnasDickus) [inline src/pages/About/About.js]: Why did you comment out the code instead of deleting? Are you expecting for it to be used somewhere else in the project?

**Review by ArnasDickus — COMMENTED** [review]
On mobile it still does not look good. Also I can't scroll to left side. on my phone.
Screenshot_20190518-222609

Screenshot_20190518-222606

File: src/pages/About/About.js (hunk 1 above)
Comment (mubarakshow) [reply src/pages/About/About.js]: True, i thought I'd be adding that block later. I'll delete it

**Comment by mubarakshow** [comment]
> On mobile it still does not look good. Also I can't scroll to left side. on my phone.

Yes I know and that was the first thing i mentioned in the comments ```1. Page not fully responsive```. I'll fix it. @ArnasDickus

**Comment by ArnasDickus** [comment]
@mubarakshow Regarding links are not redirecting externally. You simple need to change www.linkedin.com to full link https://www.linkedin.com/. I didn't said anything because I thought this is just a placeholder and not supposed to work.

Also [76](https://github.com/zero-to-mastery/mappypals/pull/76) pull request was merged.

**Comment by mubarakshow** [comment]
> @mubarakshow Regarding links are not redirecting externally. You simple need to change [www.linkedin.com](http://www.linkedin.com) to full link https://www.linkedin.com/. I didn't said anything because I thought this is just a placeholder and not supposed to work.
> 
> Also [76](https://github.com/zero-to-mastery/mappypals/pull/76) pull request was merged.

The links are placeholders for now. I'll get everyone to add their proper details later. Also, i'll try the suggested fix. Thanks!

**Comment by ArnasDickus** [comment]
I try to review it.

**Comment by ArnasDickus** [comment]
@mubarakshow So your about page seems fine. However do you have any idea why About us, Contact Us, footer on navbar need to be hovered in order to see text? It isn't the case in dev navbar.

**Comment by mubarakshow** [comment]
@ArnasDickus I didn't tamper with the navbar but I can take a closer look at it.

**Comment by ArnasDickus** [comment]
@mubarakshow you probably tampered indirectly with navbar. Because this is how dev version looks like.
navbar-dev

**Comment by mubarakshow** [comment]
I've fixed the navbar issue you raised @ArnasDickus

**Comment by ArnasDickus** [comment]
I try to review it.

**Comment by ArnasDickus** [comment]
Alright I don't see any further issues. Good work. Going to merge it now.

**Pull request merged**